source/deployment/mobile-app-deployment.rst

```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
-
-AppConfig (Supported) vs. App Wrapping (Not Supported)
+AppConfig (supported) vs. app wrapping (not supported)
 ======================================================
 
     **Note:** Mattermost only supports the AppConfig standard. It does not support app wrapping. Use app wrapping at your own risks.
 
-So app wrapping isn’t supported, and the AppConfig standard is, but what is the difference between the two?
-
-While there are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining both, let’s take a look at AppConfig first.
+There are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining AppConfig and app wrapping, let us take a look at AppConfig first.
 
 AppConfig
 ---------
 
-“A community focused on providing tools and best practices around native capabilities in mobile operating systems to enable a more consistent, open and simple way to configure and secure mobile apps in order to increase mobile adoption in business.” - AppConfig.org
+"A community focused on providing tools and best practices around native capabilities in mobile operating systems to enable a more consistent, open and simple way to configure and secure mobile apps in order to increase mobile adoption in business." - AppConfig.org
 
 So that sounds great, but are there any benefits for users?
 
-Again, in the AppConfig Community’s words, “Users benefit with instant mobile productivity and a seamless out-of-the box experience, and businesses benefit with secure work-ready apps with minimal setup required while leveraging existing investments in Enterprise Mobility Management (EMM), VPN, and identity solutions. Put another way, your apps are simpler to configure, secure and deploy.”
+Again, in the AppConfig Community’s words, "Users benefit with instant mobile productivity and a seamless out-of-the box experience, and businesses benefit with secure work-ready apps with minimal setup required while leveraging existing investments in Enterprise Mobility Management (EMM), VPN, and identity solutions. Put another way, your apps are simpler to configure, secure and deploy."
 
-For now, focus on that last part, “... your apps are simpler to configure, secure and deploy.” AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
+For now, focus on that last part, "... your apps are simpler to configure, secure and deploy." AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
 
```

Comment:
Wondering if the three quotes mentioned above should have links to the article in question? Or are they all from: https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html?

Reply:
That is a great question! AFAIK they are all direct quotes from [appconfig.org](https://www.appconfig.org)

Reply:
Might be helpful to link to that article or specific section in the article to provide content for the reader? 3/5; not sure what the standard for quotes is in CE docs. :)

Reply:
No, I mean it's literally on appconfig.com, right on their front page.

![appconfig](https://user-images.githubusercontent.com/11317357/84890366-99341180-b09a-11ea-8b16-35b60dee8e16.png)

Reply:
Great. :)